docs/notebooks/example.ipynb

```diff
@@ -11,9 +11,9 @@
     "\n",
```

Comment:
<p>I think the preferred <code>thresh </code>units would be <code>degC</code> even if xclim knows what to do here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>